### PR TITLE
[CTOR-514] [Doc] New Pack RRDCached

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Initial pack release for this plugin already existing.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+
+Le connecteur apporte le modèle de service suivant
+(classé selon le modèle d'hôte auquel il est rattaché) :
+
+<Tabs groupId="sync">
+<TabItem value="App-Rrdcached-custom" label="App-Rrdcached-custom">
+
+| Alias | Modèle de service          | Description                              |
+|:------|:---------------------------|:-----------------------------------------|
+| Stats | App-Rrdcached-Stats-custom | Check Updates cache of rrdcached daemon  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md:24]: - unclear
- EN in FR

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+|:---------------|:------|
+| UpdatesWaiting | count |
+| QueueLenght    | coun  |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+Pour utiliser ce connecteur, vous devez installer et configurer RRDCached (voir cette [documentation](https://oss.oetiker.ch/rrdtool/doc/rrdcached.en.html)).
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md:48]:
```suggestion
Pour utiliser ce connecteur, vous devez [installer et configurer RRDCached](https://oss.oetiker.ch/rrdtool/doc/rrdcached.en.html).
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+
+1. Si vous avez utilisé un modèle d'hôte et coché la case **Créer aussi les services liés aux modèles**, les services associés au modèle ont été créés automatiquement, avec les modèles de services correspondants. Sinon, [créez les services désirés manuellement](/docs/monitoring/basic-objects/services) et appliquez-leur un modèle de service.
+2. Renseignez les macros désirées (par exemple, ajustez les seuils d'alerte). Les macros indiquées ci-dessous comme requises (**Obligatoire**) doivent être renseignées.
+
+<Tabs groupId="sync">
+<TabItem value="Stats" label="Stats">
+
+| Macro          | Description                                                                                                                                      | Valeur par défaut | Obligatoire |
+|:---------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| WARNINGQUEUE   | Warning number of nodes in RRDCached queue                                                                                                       | 70                |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md:164]: unclear

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+| CRITICALQUEUE  | Critical number of nodes in RRDCached queue                                                                                                      | 100               |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md:165]: unclear

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+| WARNINGUPDATE  | Warning number of cached RRD updates (One update can include several values)                                                                     | 3000              |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md:166]: unclear

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+| CRITICALUPDATE | Critical number of cached RRD updates (One update can include several values)                                                                    | 5000              |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md:167]: unclear

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 7]
```diff
@@ -0,0 +1,291 @@
...
+	--critical-update='5000' \
+	--warning-queue='70' \
+	--critical-queue='100' \
+	--verbose
+```
+
+La commande devrait retourner un message de sortie similaire à :
+
+```bash
+OK:  | 'UpdatesWaiting'=10updates;;;0;
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md:201]: is that sufficient?

File: pp/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 8]
```diff
@@ -0,0 +1,290 @@
...
+* **App-Rrdcached-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="App-Rrdcached-custom" label="App-Rrdcached-custom">
+
+| Service Alias | Service Template           | Service Description                                     |
+|:--------------|:---------------------------|:--------------------------------------------------------|
+| Stats         | App-Rrdcached-Stats-custom | Affiche les mise à jour du cache d'un daemon rrdcached  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-rrdcached.md:23]: FR in EN

Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-rrdcached.md:23]:
```suggestion
| Stats         | App-Rrdcached-Stats-custom | Affiche les mises à jour du cache d'un daemon rrdcached  |
```

File: pp/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 9]
```diff
@@ -0,0 +1,290 @@
...
+|:---------------|:------|
+| UpdatesWaiting | count |
+| QueueLenght    | count |
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+To use this connector you must have installed and configured RRDCached (see this [documentation](https://oss.oetiker.ch/rrdtool/doc/rrdcached.en.html)).
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-rrdcached.md:47]:
```suggestion
To use this connector you must have [installed and configured RRDCached](https://oss.oetiker.ch/rrdtool/doc/rrdcached.en.html).
```

File: pp/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 10]
```diff
@@ -0,0 +1,290 @@
...
+|:---------------|:---------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| WARNINGQUEUE   | Warning number of nodes in RRDCached queue                                                                                             | 70                |             |
+| CRITICALQUEUE  | Critical number of nodes in RRDCached queue                                                                                            | 100               |             |
+| WARNINGUPDATE  | Warning number of cached RRD updates (One update can include several values)                                                           | 3000              |             |
+| CRITICALUPDATE | Critical number of cached RRD updates (One update can include several values)                                                          | 5000              |             |
+| EXTRAOPTIONS   | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options). | --verbose         |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-rrdcached.md:169]: unclear

File: pp/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 11]
```diff
@@ -0,0 +1,290 @@
...
+	--critical-update='5000' \
+	--warning-queue='70' \
+	--critical-queue='100' \
+	--verbose
+```
+
+The expected command output is shown below:
+
+```bash
+OK:  | 'UpdatesWaiting'=10updates;;;0;
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-rrdcached.md:200]: is that sufficient?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 12]
```diff
@@ -0,0 +1,478 @@
...
+
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="App-Rrdcached-Tcp-custom" label="App-Rrdcached-Tcp-custom">
+
+| Alias       | Modèle de service                     | Description                              |
+|:------------|:--------------------------------------|:-----------------------------------------|
+| Ping-Socket | App-Rrdcached-Ping-Socket-Tcp-custom  | Check Updates cache of rrdcached daemon  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md:25]:
```suggestion
| Ping-Socket | App-Rrdcached-Ping-Socket-Tcp-custom  | Monitor status and performance of RRDcached daemon  |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 13]
```diff
@@ -0,0 +1,478 @@
...
+| Stats       | App-Rrdcached-Stats-Tcp-custom        | Check Updates cache of rrdcached daemon  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md:26]:
```suggestion
| Stats       | App-Rrdcached-Stats-Tcp-custom        | Monitor status and performance of RRDcached daemon  |
```

File: pp/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 14]
```diff
@@ -0,0 +1,477 @@
...
+* **App-Rrdcached-Unix-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="App-Rrdcached-Tcp-custom" label="App-Rrdcached-Tcp-custom">
+
+| Service Alias | Service Template                    | Service Description                      |
+|:--------------|:------------------------------------|:-----------------------------------------|
+| Ping-Socket | App-Rrdcached-Ping-Socket-Tcp-custom  | Check Updates cache of rrdcached daemon  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-rrdcached.md:24]:
```suggestion
| Ping-Socket | App-Rrdcached-Ping-Socket-Tcp-custom  | Monitor status and performance of RRDcached daemon  |
```

File: pp/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 15]
```diff
@@ -0,0 +1,477 @@
...
+| Stats       | App-Rrdcached-Stats-Tcp-custom        | Check Updates cache of rrdcached daemon  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-rrdcached.md:25]:
```suggestion
| Stats       | App-Rrdcached-Stats-Tcp-custom        | Monitor status and performance of RRDcached daemon  |
```

File: pp/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 16]
```diff
@@ -0,0 +1,477 @@
...
+
+> The services listed above are created automatically when the **App-Rrdcached-Tcp-custom** host template is used.
+
+</TabItem>
+<TabItem value="App-Rrdcached-Unix-custom" label="App-Rrdcached-Unix-custom">
+
+| Service Alias | Service Template                    | Service Description                      |
+|:------------|:--------------------------------------|:-----------------------------------------|
+| Ping-Socket | App-Rrdcached-Ping-Socket-Unix-custom | Check Updates cache of rrdcached daemon  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-rrdcached.md:34]:
```suggestion
| Ping-Socket | App-Rrdcached-Ping-Socket-Unix-custom | CMonitor status and performance of RRDcached daemon |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 17]
```diff
@@ -0,0 +1,478 @@
...
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **App-Rrdcached-Tcp-custom** est utilisé.
+
+</TabItem>
+<TabItem value="App-Rrdcached-Unix-custom" label="App-Rrdcached-Unix-custom">
+
+| Alias       | Modèle de service                     | Description                              |
+|:------------|:--------------------------------------|:-----------------------------------------|
+| Ping-Socket | App-Rrdcached-Ping-Socket-Unix-custom | Check Updates cache of rrdcached daemon  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md:35]:
```suggestion
| Ping-Socket | App-Rrdcached-Ping-Socket-Unix-custom | Monitor status and performance of RRDcached daemon |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md [hunk 18]
```diff
@@ -0,0 +1,478 @@
...
+| Stats       | App-Rrdcached-Stats-Unix-custom       | Check Updates cache of rrdcached daemon  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rrdcached.md:36]:
```suggestion
| Stats       | App-Rrdcached-Stats-Unix-custom       | Monitor status and performance of RRDcached daemon |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**